src/layouts/EditNavBar.jsx

```diff
@@ -612,25 +621,47 @@ const EditNavBar = ({ match }) => {
 
   return (
     <>
-      {showDeletedText && !isEmpty(deletedLinks) && (
+      {!isEmpty(deletedLinks) && (
         <GenericWarningModal
+          isOpen={isRemovedOpen}
+          onClose={onRemovedClose}
           displayTitle="Removed content"
           displayText={`Some of your content has been removed as they attempt to link to invalid folders. No changes are permanent unless you press Save on the next page.<br/>${deletedLinks}`}
```

Comment:
we could actually skip the `parse` call if this accepts a `Text` as its children

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/941/commits/bae8cb2823b64f92ac2e40cee947dd907a0baae9